### PR TITLE
Extract message interpolation

### PR DESCRIPTION
**Pull request by RobCherry**
Requested by https://github.com/mozilla-services/heka/pull/1420#discussion_r26525940

### CODE REVIEW

**Comment by trink** [comment]
The cbuf_utlis module was rejected here: https://github.com/mozilla-services/heka/pull/1365#issuecomment-76038929

**Comment by RobCherry** [comment]
Fixed.  Sorry about that, I had previously read the comment about `cbuf_utils` in #1365 but I had included it so I could pull in 2250129 without conflicts.

File: sandbox/lua/encoders/cbuf_librato.lua [hunk 1]
```diff
@@ -29,6 +32,8 @@ Config:
     type = "SandboxEncoder"
     filename = "lua_encoders/cbuf_librato.lua"
     preserve_data = true
+      [cbuf_librato_encoder.config]
+      message_key = "%{Logger}:&{Hostname}:%{payload_name}"
```
Comment (rafrombrc) [inline sandbox/lua/encoders/cbuf_librato.lua]: Typo `&{Hostname}` -> `%{Hostname}`

Comment (rafrombrc) [inline sandbox/lua/encoders/cbuf_librato.lua]: Ignore, I'll fix this and merge...

**Pull request merged**